test/com/facebook/buck/artifact_cache/DirArtifactCacheTest.java

```diff
@@ -300,9 +300,12 @@ public void testCacheStoresAndFetchHits() throws IOException {
         .map(input -> input.getFileName().toString())
         .collect(MoreCollectors.toImmutableSet());
 
+    /* Remove comparisson that not used, as it triggers CheckReturnValue
+     * (see http://errorprone.info/bugpattern/CheckReturnValue)
+     */
     for (RuleKey ruleKey : ImmutableSet.of(ruleKeyX, ruleKeyY, ruleKeyZ)) {
-      filenames.contains(ruleKey.toString());
-      filenames.contains(ruleKey.toString() + ".metadata");
+      assertTrue(filenames.contains(ruleKey.toString()));
```

Comment:
Oh, nice, when i actually run the test (i only compiled the tests before pushing the PR), I'm seein gactually this breakage:

```
RESULTS FOR //test/com/facebook/buck/artifact_cache:artifact_cache
PASS    <100ms  3 Passed   0 Skipped   0 Failed   com.facebook.buck.artifact_cache.AbstractNetworkCacheTest
PASS    <100ms 16 Passed   0 Skipped   0 Failed   com.facebook.buck.artifact_cache.ArtifactCacheBuckConfigTest
PASS     179ms  4 Passed   0 Skipped   0 Failed   com.facebook.buck.artifact_cache.ArtifactCachesTest
FAIL     307ms 14 Passed   0 Skipped   1 Failed   com.facebook.buck.artifact_cache.DirArtifactCacheTest
FAILURE com.facebook.buck.artifact_cache.DirArtifactCacheTest testCacheStoresAndFetchHits: 
Expected: iterable containing ["ec40f9121202be3508b4f5ab7b0a511d3bdcf9d9"]
     but: item 0: was "8e2eabf8408715a938eaf41c6933ab0da79da6da"
java.lang.AssertionError: 
Expected: iterable containing ["ec40f9121202be3508b4f5ab7b0a511d3bdcf9d9"]
     but: item 0: was "8e2eabf8408715a938eaf41c6933ab0da79da6da"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at com.facebook.buck.artifact_cache.DirArtifactCacheTest.testCacheStoresAndFetchHits(DirArtifactCacheTest.java:304)
```

Ws this the reason, why assert werenÄt implemened?